apps/docs/src/app/core/component-docs/table/table-docs.component.ts

```diff
@@ -55,6 +74,136 @@ export class TableDocsComponent {
         }
     ];
 
+    tableWithoutBordersExample: ExampleFile[] = [
+        {
+            language: 'html',
+            code: tableWithoutBordersHtmlSrc,
+            fileName: 'table-without-borders-example'
+        },
+        {
+            language: 'typescript',
+            component: 'TableExampleComponent',
+            code: tableWithoutBordersTsSrc,
+            fileName: 'table-without-borders-example'
+        }
+    ];
+
+    tableFooterExample: ExampleFile[] = [
+        {
+            language: 'html',
+            code: tableFooterHtml,
+            fileName: 'table-example'
+        },
+        {
+            language: 'typescript',
+            component: 'TableFooterExampleComponent',
+            code: tableFooterTs,
+            fileName: 'table-footer-example'
+        }
+    ];
+
+    tablePopInExample: ExampleFile[] = [
+        {
+            language: 'html',
+            code: tablePopInHtml,
+            fileName: 'table-popin-example'
+        },
+        {
+            language: 'typescript',
+            component: 'TablePopinExampleComponent',
+            code: tablePopInHTs,
+            fileName: 'table-popin-example'
+        }
+    ];
+
+    tableCustomColumnsExample: ExampleFile[] = [
+        {
+            language: 'html',
+            code: tableCustomHtml,
+            fileName: 'table-custom-columns-example'
+        },
+        {
+            language: 'typescript',
+            code: tableCustomTs,
+            fileName: 'table-custom-columns-example',
+            component: 'TableCustomColumnsExampleComponent',
+            name: 'Table Component'
+        },
+        {
+            language: 'typescript',
+            code: tableDialogCustom,
+            fileName: 'table-custom-dialog',
+            component: 'TableCustomDialogComponent',
+            name: 'Dialog Component'
+        },
+        {
+            language: 'typescript',
+            code: tableFilterPipe,
+            fileName: 'table-filter',
+            component: 'TableFilterPipe',
+            pipe: true,
+            name: 'Filter Pipe'
+        }
+    ];
+
+    tableColumnSortExample: ExampleFile[] = [
+        {
+            language: 'html',
+            code: tableColumnSortHtmlSrc,
+            fileName: 'table-column-sorting-example'
+        },
+        {
+            language: 'typescript',
+            component: 'TableColumnSortExampleComponent',
```

Comment:
Change component name to `TableColumnSortingExampleComponent`, Current one doesn't work on stackblitz